### ISSUE TITLE
Skyrim VR - OCULUS - input emulator

### ISSUE THREAD

**Issue by jessesomogyi** [matzman666/OpenVR-InputEmulator#94]
Hello MATZMANN,
     Your control emulator for Fall Out 4 OCULUS works great - excellent. I enjoyed over 14hrs playing.  I just installed NEW Skyrim VR Steam Oculus and it crashes upon starting.  Could the input-emulator cause this? I cannot see an easy way to uninstall the input emulator. Maybe that would be a solution (maybe worth a try?) I was once told - It never hurts to ask.  _ I will await your response.  -Jesse

**Comment by v1c3q4** [comment]
Yeah open vr emulator is great for FO VR but now in order to play Skyrim VR we need to opt out of beta in Seteam VR which disables the emulator of course. Please help these are my 2 favorite VR games.

**Comment by TheTruckinGamerVR** [comment]
Same here want to play skyrim amd fallout without jumping through hoops and now i can get ovrie to work at all even on the beta branch for it. Keeps crashing. Please help
